packages/driver/cypress/integration/e2e/multi-domain/logging.spec.ts

```diff
@@ -0,0 +1,146 @@
+describe('cy.origin logging', () => {
+  const { _ } = Cypress
+
+  it('groups callback commands on a passing test', () => {
+    const logs: any[] = []
```

Comment:
any reason `logs` is defined per test? Can it be defined up top and cleared in hooks or is that causing pollution in the `logs` array?

Reply:
I think it's better to have it scoped per test, then there's no need to worry about clearing it out or there possibly being pollution between tests. But maybe I'm missing something? What would be the advantage of scoping it higher?

Reply:
Just duplication and the convention we are currently following in a lot of the `log` tests, but I think this option is probably safer. Don't think it matters too much 😄